test/e2e/e2e_utils_test.go

```diff
@@ -239,6 +263,20 @@ func RemoveFile(fullPath string) error {
 	return nil
 }
 
+func defaultUserAuth() (*auth.User, error) {
+	tokenVal, err := getToken(defaultUsername, defaultOrganization)
+	if err != nil {
+		return nil, fmt.Errorf("unable to create user: %v", err)
+	}
+	token := jwt.New(jwt.SigningMethodRS256)
+	token.Raw = tokenVal
+	return &auth.User{
+		Username:     defaultUsername,
```

Comment:
I think you don't need the username and orgnization at all anymore, no?
You can just return token here